scripts/sync_codemirror.sh

```diff
@@ -44,11 +42,17 @@ excluded_dirs=".github .circleci"
 # Go to the root of the repo
 cd "$(git rev-parse --show-cdup)" || exit 1
 
-source_dir="$(pwd)/${source_path}"
-
 tmp_dir="$(mktemp -d)"
 trap 'rm -rf "${tmp_dir}"' EXIT
 
+source_dir() {
```

Comment:
Why is

Reply:
since you want to manage multiple different mirroring you need a way to get the `source_dir`. But you definitely don't want to have a global value for that, since you will have different `source_dir`